pipeline/test_email_notifier.py

```diff
@@ -82,9 +83,7 @@ def test_send_email_success():
 
 
 def test_send_email_failure():
```

Comment:
Can you add a docstring:
""""Test that an error is logged when the email fails to send."""